documentation/staging/content/userguide/base-images/patch-images.md

```diff
@@ -0,0 +1,173 @@
+(see [Auxiliary images]({{< relref "/userguide/managing-domains/model-in-image/auxiliary-images.md" >}})).
+The operator will then perform a [rolling restart]({{< relref "/userguide/managing-domains/domain-lifecycle/restarting#overview" >}})
+of the WebLogic domain to update the Oracle Home of the servers.
+
+#### Model in Image without auxiliary images
+
+For Model in Image domains _without_ using auxiliary images:
+
+- The container image contains the JDK, WebLogic Server binaries,
+  a [WebLogic Deployment Tooling](https://oracle.github.io/weblogic-deploy-tooling/) (WDT) installation,
+  WDT model files, WDT variable files, and an application archive file.
+- The domain home is generated by the operator during runtime.
+
+If you need to update the image for a running Model in Image domain,
+then simply follow the same steps that were used to create the original
+image as described in [Create a custom image with patches applied](#create-a-custom-image-with-patches-applied),
+and edit the domain resource's `domain.spec.image` attribute
+with the new image's name/tag (`mydomain:v2`).
+The operator will then perform a [rolling restart]({{< relref "/userguide/managing-domains/domain-lifecycle/restarting#overview" >}})
+of the WebLogic domain to update the Oracle Home of the servers.
+
+#### Domain in Image
+
+If you need to update the image for a running Domain in Image domain,
+then use the WIT [`rebase`](https://oracle.github.io/weblogic-image-tool/userguide/tools/rebase-image/)
+command to update the Oracle Home
+for an existing domain image using the patched Oracle Home from a patched container image.
+For Domain in Image domains:
+
+- The container image contains the JDK, WebLogic Server binaries, and domain home.
+
+- The domain home is generated during image creation using either WLST or WDT,
+  usually with the assistance of the WebLogic Image Tool (WIT).
+
+The `rebase` command does the following:
+
+- Minimizes the image size. The alternative `update` command does _not_ remove old WebLogic installations
+  in the image but instead, layers new WebLogic installations on top of the original installation, thereby
+  greatly increasing the image size; we strongly recommend against using the `update` command in this situation.
+
+- Creates a new WebLogic image by copying an existing WebLogic domain home
+  from an existing image to a new image.
+  It finds the domain home location within the original image
+  using the image's internal `DOMAIN_HOME` environment variable.
+
+- Maintains the same security configuration
+  as the original image because the domain home is copied
+  (for example, the `DOMAIN_HOME/security/SerializedSystemIni.dat` file).
+  This ensures that pods that are based on the new image
+  are capable of joining an already running
+  domain with pods on an older version of the image with same security configuration.
+
+Using `rebase`, the new image can be created in one of two ways:
+
```

Comment:
I think for Rebase we need to repeat the message to either apply security patches or use a patched WebLogic Image

Reply:
Got it. Will appear in my next PR.